src/jobs/promote_latest_package_build.yml

```diff
@@ -0,0 +1,49 @@
+description: |
+  This job promotes the the latest release candidate package version.
+
+parameters:
+  additionalSteps:
+    default: []
+    description: Provide optional steps you would like to run prior to the package promote
+    type: steps
+  devhubUsername:
+    description: Username or alias of the DevHub
+    type: string
+  devhubInstanceUrl:
+    description: Instance url of the DevHub, if you have a custom domain enabled.
+    default: https://login.salesforce.com
+    type: string
+  devhubJwtKey:
+    description: Name of the environment variable containing the base64 encoded private JWT key (server.key file) to authenticate to the DevHub.
+    default: DEVHUB_JWT_KEY
+    type: env_var_name
+  devhubConsumerKey:
+    description: Name of the environment variable containing the consumner key of the connected app on the DevHub. The CLI refers to this as "client id"
+    default: DEVHUB_CONSUMER_KEY
+    type: env_var_name
+  packageVersionId:
+    description: Environment variable with the package version id to promote. Required if promoteLatestBuild is false. Has no effect, if it's true.
+    default: SUBSCRIBER_PACKAGE_VERSION_ID
+    type: env_var_name
+  promoteLatestBuild:
+    description: Promotes the latest release candidate build of the package version. This queries the latest build from the DevHub.
+    default: true
+    type: boolean  
```

Comment:
Überleg dir, ob der Name des Jobs (und damit auch die definierte Funktionalität, über die wir gesprochen hatten), wirklich zu den Parametern passen.

Ganz konkret: Wenn wir sagen, dass der Job "promote latest package build" heißt, darf man erwarten, dass der **letzte package build** promoted wird. Inwiefern machen in diesem Kontext die beiden Parameter Sinn?